src/sentry/static/sentry/app/views/organizationEvents/events.jsx

```diff
@@ -26,6 +29,44 @@ const parseRowFromLinks = (links, numRows) => {
   return `${currentStart}-${currentStart + numRows - 1}`;
 };
 
+class TotalEventCount extends AsyncComponent {
+  static propTypes = {
+    organization: SentryTypes.Organization.isRequired,
+    location: PropTypes.object.isRequired,
+    isAllResults: PropTypes.bool.isRequired,
+    numRows: PropTypes.number.isRequired,
+  };
+
+  getEndpoints() {
+    const {organization, location} = this.props;
+    let {statsPeriod, ...query} = location.query;
+
+    return [
+      [
+        'eventsMeta',
+        `/organizations/${organization.slug}/events-meta/`,
```

Comment:
Is the plan to keep this on a separate endpoint? It'd be nice to include meta data with the org events request

Reply:
the problem with doing that is it will slow the search results down since this would have to be a separate query. so i think probably not :-/